nccs/analysis.py

```diff
@@ -431,22 +512,27 @@ def calculate_indirect_impacts_from_df(df, io_a, config, direct_output_dir, indi
         try:
             LOGGER.info(f"Calculating indirect {io_a} impacts for {logging_dict}...")
             imp = Impact.from_hdf5(row['yearset_path'])
+
+            # Strip the crop type from the sector name, as they are not in the sector list of the MRIO table
+            impacted_sector = "agriculture" if "agriculture" in row['sector'] else row['sector']
```

Comment:
Am I right in thinking that, by this point in the analysis, we should have already recombined the different crops, meaning there's only 'agriculture' as a sector, and never 'agriculture_whe' etc?

Reply:
True, seems I have missed this leftover